### PR TITLE
#168381685: Redesigned the UI

### PR DESCRIPTION
**Pull request by Mastel22**
#### What does this PR do?
Create pull request  for redesigning the UI
#### Description of Task to be completed?
-Redesigned the User Interface
#### How should this be manually tested?
-By cloning the repo to your PC and viewing in chrome
#### What are the relevant pivotal tracker stories?
#168381685: https://www.pivotaltracker.com/story/show/168381685
#### Screenshots (if appropriate)
image
mentors

Login
signup
request form

### CODE REVIEW

File: server/middleware/token.middleware.js [hunk 1]
```diff
@@ -17,8 +17,6 @@ const tokenVerifier = async (req, res, next) => {
     const verified = jwt.verify(token, process.env.KEY);
     // const user = users.find((user) => user.email === verified.email);
```
Comment (oesukam) [inline server/middleware/token.middleware.js]: Why have you commented this line?

Comment (Mastel22) [reply server/middleware/token.middleware.js]: Over the weekend.That was relevant for challenge 2 and I was adding codes for challenge 3

Comment (oesukam) [reply server/middleware/token.middleware.js]: Can you delete it then if you are no longer using it?

Comment (Mastel22) [reply server/middleware/token.middleware.js]: Done now

**Pull request merged**